datamodel/src/test/java/com/hartwig/serve/datamodel/EvidenceDirectionTest.java

```diff
@@ -0,0 +1,16 @@
+package com.hartwig.serve.datamodel;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+public class EvidenceDirectionTest  {
+
+    @Test
+    public void canCompareEvidenceDirections() {
+        assertTrue(EvidenceDirection.RESISTANT.isHigher(EvidenceDirection.PREDICTED_RESISTANT));
+        assertTrue(EvidenceDirection.RESPONSIVE.isHigher(EvidenceDirection.PREDICTED_RESPONSIVE));
+        assertTrue(EvidenceDirection.RESPONSIVE.isHigher(EvidenceDirection.RESISTANT));
```

Comment:
Could you also add some tests for `NO_BENEFIT`? In my mind "responsive > no_benefit > resistant" but not sure this is captured by your logic.

Reply:
it should be response -> predicted responsive > resistent > predicted resistent > no benefit 
also good to know, no benefit should always not reportable (so this hasn't any effect downstream). Or do I make a misinterpretation?

Reply:
I'd have to think a bit more about it, but my first thought is: if you can choose between 2 treatments you want to pick the treatment with "best" direction. In this case, it's better to take a drug with NO_BENEFIT rather than a drug with "RESISTANCE". Although at the same time, both drugs don't work so maybe I am wrong.

I do think at least it's better to take a drug with PREDICTED_RESISTANCE, since there is a chance that the resistance is wrong and the drug may still give benefit? 

We could also decide to not rely at all on the datamodel providing an ordering, maybe its a type of data that doesn't have a very logical ordering?